python/MetrologyData.py

```diff
@@ -296,6 +299,75 @@ def _xyz(self, line):
         data = [float(x) for x in line.split()[:3]]
         return data[0], data[1], 1e3*data[2]
 
+class Ts5Data(MetrologyData):
+    """
+    Abstraction for raft (RSA and RTM) metrology scans with TS5.
+    """
+    def __init__(self, infile):
+        super(Ts5Data, self).__init__(infile)
+
+    def _read_data(self):
+        # The TS5 metrology data are in csv files.  The first three entries on each
+        # line are commanded x and y and the measured (summed) z, in mm.
+        # Here to allow option of differencing two data sets, infile is
+        # assumed to be a list of 1 or 2 elements
+        data = dict([(key, []) for key in 'XYZ'])
+        
+        # Test to see whether a single string or a list of two files has been
+        # passed for infile
+        if type(self.infile) == type('abc'):
```

Comment:
Probably better to use `isinstance(self.infile, str)` instead of `type(self.infile) == type('abc')`